src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/JJDispute.java

```diff
@@ -295,7 +295,16 @@ public class JJDispute extends Auditable<String> {
 
 	@Column
 	@Schema(nullable = true)
+	@Enumerated(EnumType.STRING)
```

Comment:
I don't think this will work. ContactType is a ShortNamedEnum and as such should be mapped using short code, not the full name of the enum (which is what this @Enumerated annotation does).

Reply:
done thanks.